<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>496</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>One Video</string>
     </attribute>
     <widget class="QLineEdit" name="singleUrl">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Youtube URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="singleSave">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Save Location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="singleBrowse">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="singleProgress">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>521</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="singleDownload">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="singleQuality">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Quality</string>
      </property>
     </widget>
     <widget class="QComboBox" name="singleCombo">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="singleGet">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="singleTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Full Playlist</string>
     </attribute>
     <widget class="QLineEdit" name="playlistUrl">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Playlist URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playlistSave">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Save Location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playlistBrowse">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="playlistProgress">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>521</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="playlistDownload">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentVideo">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="fullPlaylist">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Full Playlist</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="currentLcd">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fullLcd">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="playlistTime">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
